extension/lib/main.js

```diff
@@ -6,11 +6,11 @@
 
```

Comment:
It would be nice to write a test for the GUI.

For example provide an API to allow tests
 * catch `frame`'s `attach` and `load` events
 * query / listen `frame`'s data: memory, gc, cc updates
 * catch `toolbar`s `attach` and `load` events

These tests will ensure that the extension has a functional GUI, it is visible, and it is working.